app/admin/store_products.rb

```diff
@@ -1,10 +1,11 @@
 ActiveAdmin.register Product do
   menu parent: I18n.t('activeadmin.titles.stores')
 
-  permit_params :store_id, :name, :price, :link, :email, :gender, :age, :novelty, :status
+  permit_params :store_id, :name, :category_id, :price, :link, :email, :gender, :age, :novelty, :status
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Layout/LineLength: Line is too long. [103/100]